com.unity.multiplayer.mlapi/Editor/NetworkedAnimatorEditor.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
-using MLAPI.Prototyping;
-using UnityEditor.Animations;
-using UnityEngine;
+//using System;
+//using MLAPI.Prototyping;
+//using UnityEditor.Animations;
+//using UnityEngine;
```

Comment:
Do we want to leave in commented-out `using` directives?

Reply:
I'll revert...the larger issue is to hop on https://jira.unity3d.com/browse/MTT-515 and bring this back.